Controller/Adminhtml/Exportcategory/Export.php

```diff
@@ -49,48 +50,103 @@ public function __construct(
         \Magento\Framework\Controller\Result\RawFactory $resultRawFactory,
         \Magento\Framework\App\Response\Http\FileFactory $fileFactory,
         \Magento\Backend\App\Action\Context $context
-    ) {
-    
+    )
+    {
```

Comment:
Please revert back to `) {` to comply with [PSR-2 standards](https://www.php-fig.org/psr/psr-2/#44-method-arguments) regarding method arguments.

Reply:
Or not...changed it